Assets/AppCenter/Plugins/AppCenterSDK/Analytics/Android/WrapperPropertyConfiguratorInternal.cs

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+//
+// Licensed under the MIT license.
+
+#if UNITY_ANDROID && !UNITY_EDITOR
+using System.Collections.Generic;
+using UnityEngine;
+using Microsoft.AppCenter.Unity.Analytics.Internal;
+using Microsoft.AppCenter.Unity.Internal.Utility;
+
+namespace Microsoft.AppCenter.Unity.Analytics
+{
+    public class WrapperPropertyConfiguratorInternal
+    {
+        public static void SetAppName(UnityEngine.AndroidJavaObject propertyConfigurator, string appName)
```

Comment:
Please use just 'AndroidJavaObject' here, without 'UnityEngine' namespace here and in other similar places